<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWindow{border-image: url(:/images/backgroud.png);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="centerWidget">
     <property name="styleSheet">
      <string notr="true">QWidget#centerWidget{border-image: url(:/images/backgroud.png);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="hrightSpace">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="detailWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>576</width>
          <height>440</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>576</width>
          <height>440</height>
         </size>
        </property>
        <widget class="QLabel" name="modeLabel">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;黑体&quot;;
color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;部署模式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="desLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>311</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;您已经进入部署模式，可以执行以下操作:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="modeWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>576</width>
           <height>211</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>576</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>576</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#modeWidget
{
border-image:url(:/images/nice_light.png) 10 10 10 10;
border-top: 10px transparent;

border-bottom: 10px transparent;

}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="installButton">
            <property name="minimumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;黑体&quot;;
border-image:url(:/images/nice_black.png) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;

QPushButton#installButton:pressed
{
        border-image:url(:/images/nice_black.png) 10 10 10 10;
        border-top: 10px transparent;
        border-right: 10px transparent;
        border-bottom: 10px transparent;
        border-left: 10px transparent;
}
</string>
            </property>
            <property name="text">
             <string>安装客户端(此操作将格式化终端硬盘)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="minimumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;黑体&quot;;
border-image:url(:/images/nice_light.pngg) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>将本地系统作为模板上传</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rebootButton">
            <property name="minimumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>576</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;黑体&quot;;
border-image:url(:/images/nice_light.pngg) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>安全弹出U盘并重启</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="hleftSpace">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lineLabel">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/line.png);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{border-image: url(:/images/backgroud.png);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="f2Label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>[F2]终端配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="arrowLabel">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;[↑][↓]选择选项&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="enterLabel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>[Space]选择确认</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hcenSpace">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="netWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="netpoint_label">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="netstate_label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;在线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ipWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>185</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="neticon_label">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>19</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ip_label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;192.168.255.255&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>  00:00
2000/01/01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
